File: python/5.py
n = 2520
resolution = 0

def evenlyDivisible(n):
  i = 2
  while i <= 20:
    if (n % i != 0):
      return True
    i += 1


while resolution < 1:
  if evenlyDivisible(n) == True:
    n += 1
  else:
    resolution = n

print(resolution)

File: js/5.js
let resolution;
let n = 1;
evenlyDivisible = n => {
  for (let i = 2; i <= 20; i++) {
    if (n % i !== 0) return true;
  }
};
while (!resolution) {
  if (evenlyDivisible(n)) {
    n++;
  } else {
    resolution = n;
  }
}
console.log(resolution);
